### PR TITLE
Mobile version

### PR DESCRIPTION
**Pull request by jr-cast**
In this next iteration of this project:

- Section about "Recent Works" was added, consisting of container allocating cards within it.
- 5 cards were styled and arranged with flexbox
- Grid system was used to accommodate cards within the card container.
- Styles based on https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988
- The README file was updated

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -52,9 +52,108 @@ <h2>I'm a software developer</h2>
...
+            <h2 id="works">My Recent works</h2>
+            <img src="Indicator.png" alt="">
+            <div class="cardContainer">
+
+                <div class="card1">
+                    <div class="img">
+                        <img class="img1" src="static/card1.jpg" alt="">
+                        <div class="cardFooter">
+                            <div class="footerTitle">
+                                <p>Multi-Post Stories Gain+Glory</p>
```
Comment (AnselemOdims) [inline index.html:68]: - [ ] Please kindly replace the paragraph `p` tag with a headings tag `h2-h6` for semantic reasons. Kindly also do this for subsequent ones

File: index.html [hunk 2]
```diff
@@ -52,9 +52,108 @@ <h2>I'm a software developer</h2>
...
+                                <li>HTML</li>
+                            </ul>
+                            <button class="cardBtn">
+                                See Project
+                            </button>
+                        </div>
+                    </div>
+                </div>
+                <div id="background2">
+                    <section id="aboutMe">
```
Comment (AnselemOdims) [inline index.html:143]: - [ ] Please kindly ensure that the background images for this section are displayed correctly and as per original design. Kindly compare below

### Your Design
image

### Original Design
image

File: index.html [hunk 3]
```diff
@@ -52,9 +52,108 @@ <h2>I'm a software developer</h2>
...
+                            If you like what you see and have a project you need coded,
+                            don’t hestiate to contact me.</p>
+                        <button id="getMyResume">
+                            Get My Resume
+                        </button>
+                    </section>
+                </div>
+
+            </div>
+        </section>
```
Comment (AnselemOdims) [inline index.html:156]: - [ ] The languages, frameworks, and skills cards are not available at the moment. Please kindly add the cards as they are part of the about section.

image

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @jr-cast, 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by jr-cast** [comment]
Addressed comments on #2:
image

File: index.html (hunk 1 above)
Comment (Kingstalux) [reply index.html]: resolved 👍

File: index.html (hunk 2 above)
Comment (Kingstalux) [reply index.html]: resolved 👍

File: index.html (hunk 3 above)
Comment (Kingstalux) [reply index.html]: resolved 👍

**Review by Kingstalux — APPROVED** [review]
Hi @jr-cast,
# Status: Approved ✔️✔️✔️
  
128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c


Good job implementing the required changes 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**